README.md

```diff
@@ -51,6 +51,10 @@ The `smdebug` library runs on Python 3. Install using the following command:
 pip install smdebug
 ```
 
+To find the latest release of Amazon SageMaker Debugger and smdebug, see the following PyPI pages.
+- [Amazon SageMaker Python SDK PyPI](https://pypi.org/project/sagemaker/)
```

Comment:
Why do we need a link to the sagemaker binary?

Reply:
It's because we have literature and examples for both sagemaker and smdebug.

Reply:
It's because we have literature and examples for both sagemaker and smdebug.

> Why do we need a link to the sagemaker binary?